rtl/include/eclhelper.hpp

```diff
@@ -1022,15 +1022,15 @@ enum ActivityInterfaceEnum
     TAImergearg_1,
```

Comment:
Can this whole enumeration be deleted?  I suspect it is no longer used.

Reply:
Yes I think so

Reply:
Fixed